lua/lspconfig/ansiblels.lua

```diff
@@ -0,0 +1,40 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+local server_name = 'ansiblels'
+
+configs[server_name] = {
+  default_config = {
+    cmd = { 'ansible-language-server', '--stdio' },
+    settings = {
+      ansible = {
+        python = {
+          interpreterPath = 'python',
+        },
+        ansibleLint = {
+          path = 'ansible-lint',
+          enabled = true,
+        },
+        ansible = {
+          path = 'ansible',
+        },
+      },
+    },
+    filetypes = { 'yaml' },
+    root_dir = function(fname)
+      return util.root_pattern '*.yml'(fname) or util.root_pattern '*.yaml'(fname)
```

Comment:
Do you want to first search up the directory tree for yml, and then search up the directory tree for yaml, or do you want to search both at each level? util.root_pattern can take a list if you mean the latter.

Reply:
I didn't know about this. I set it to do the latter.